packages/tools/Makefile

```
@@ -1358,20 +1360,47 @@ remote-install-root-aws-credentials:
 	@if [ ! -z "${STORAGE_ACCESS_KEY}" -a ! -z "${STORAGE_SECRET_KEY}" ];then\
 		H=$$(tr -d '\r\n[:space:]' < ${CLOUD_HOST_FILE});\
 		U=$$(tr -d '\r\n[:space:]' < ${CLOUD_USER_FILE});\
-		ssh ${SSHOPTS} $$U@$$H "sudo mkdir /root/.aws";\
+		ssh ${SSHOPTS} $$U@$$H "sudo mkdir -p /root/.aws";\
 		cat ${TOOLS_PATH}/.aws/config | ssh ${SSHOPTS} $$U@$$H "sudo tee /root/.aws/config > /dev/null";\
 		cat ${TOOLS_PATH}/.aws/credentials | sed "s/<AWS_ACCESS_KEY_ID>/${STORAGE_ACCESS_KEY}/;s/<AWS_SECRET_ACCESS_KEY>/${STORAGE_SECRET_KEY}/;" | ssh ${SSHOPTS} $$U@$$H "sudo tee /root/.aws/credentials > /dev/null";\
 	fi
 
-remote-install-monitor: remote-install-root-aws-credentials
+remote-install-monitor: remote-install-monitor-start remote-install-monitor-wait
```

checkmake (0.3.2)

[warning] Target "remote-install-monitor" should be declared PHONY.

(phonydeclared)